apps/web/src/pages/Users/UserEmployeeInformationPage/components/CareerObjectiveSection.tsx

```diff
@@ -125,9 +125,15 @@ const CareerObjectiveSection = ({
               data-h2-margin-bottom="base:selectors[>li:not(:last-child)](x.125)"
               data-h2-padding-left="base(x1)"
             >
-              {employeeProfile.careerObjectiveWorkStreams.map((workStream) => (
-                <li key={workStream.id}>{workStream?.name?.localized}</li>
-              ))}
+              {employeeProfile.careerObjectiveWorkStreams
+                .sort((a, b) =>
```

Comment:
Using sort in a fluent chain is a little risky since it's an "in-place" operation.  Maybe it makes sense to use the new "toSorted" instead?

Reply:
Good point.  Yeah, that's a good solution, too.  :+1:

Reply:
7e44323d1ee3a3f90b33760c3d2f81af90a84945

Reply:
Looks good, thanks.